contents/docs/integrate/vercel-marketplace.mdx

```diff
@@ -0,0 +1,100 @@
+---
+title: Vercel Marketplace Integration
+sidebarTitle: Vercel Marketplace
+icon: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/vercel_icon_svgrepo_com_b7e78b41f9.svg
+---
+
+import { CalloutBox } from 'components/Docs/CalloutBox'
+
+The Vercel Marketplace integration adds PostHog feature flags and experiments to your Vercel projects with billing handled through Vercel.
+
+Choose this integration if you do not already have a PostHog account or you prefer to consolidate payment for PostHog inside your Vercel invoice.
+
+## Installing the integration
+
+1. Go to the [PostHog integration](https://vercel.com/integrations/posthog) on Vercel Marketplace
+2. Click **Install**
+
+This creates a PostHog organization and project, and adds environment variables to your Vercel project.
+
+## Environment variables
+
+The integration adds:
+- `POSTHOG_PROJECT_API_KEY` - Your PostHog project API key
+- `POSTHOG_HOST` - PostHog API host URL
+
+These names work across all frameworks. For client-side frameworks, rename them in your Vercel project settings:
+
+| Framework | Rename to |
+|-----------|-----------|
+| Next.js | `NEXT_PUBLIC_POSTHOG_KEY`, `NEXT_PUBLIC_POSTHOG_HOST` |
+| Vite/SvelteKit | `VITE_POSTHOG_KEY`, `VITE_POSTHOG_HOST` |
+| Nuxt | `NUXT_PUBLIC_POSTHOG_KEY`, `NUXT_PUBLIC_POSTHOG_HOST` |
+
+Server-side code can use the original names.
+
+## Installing the SDK
+
+Install PostHog in your application using your framework's guide:
+
+- [Next.js](/docs/libraries/next-js)
+- [React](/docs/libraries/react)
+- [Svelte](/docs/libraries/svelte)
+- [All frameworks](/docs/libraries)
+
+When initializing, disable products not included in the integration:
+
+```tsx
+posthog.init(process.env.NEXT_PUBLIC_POSTHOG_KEY!, {
+  api_host: process.env.NEXT_PUBLIC_POSTHOG_HOST,
+  autocapture: false,
+  disable_session_recording: true
+})
+```
+
+Then use feature flags:
+
+```tsx
+import { useFeatureFlagEnabled } from 'posthog-js/react'
+
+export function NewFeature() {
+  const enabled = useFeatureFlagEnabled('new-checkout')
+  return enabled ? <NewCheckout /> : <OldCheckout />
+}
+```
+
+## Creating feature flags and experiments
+
+Access the PostHog dashboard from your Vercel project's **Integrations** tab.
```

Comment:
Link to their integrations tab

Reply:
Can also add a direct link to their page in PostHog. You can use `app.posthog.com` to automatically redirect between `eu` and `us`. You can omit `/project/id` and we'll automatically add it as well